### DISCUSSION TITLE
CMake for BaseTools?

### DISCUSSION THREAD

**Discussion by Elemeants** [tianocore/edk2#3578] [Ideas]
Hi!

As the title says, I was wondering if it is worth to implement a CMake structure to build BaseTools.
I do a quick experiment and I was able to build CMake.

Fork branch with my changes: [edk2-fork](https://github.com/Elemeants/edk2/tree/dev/Elemeants/cmake-base-tools)
![image](https://user-images.githubusercontent.com/33843865/199379507-c97ef12b-a912-4445-b411-8bf9bf474ccf.png)

**Comment by lersi** [comment]
it is not worth it, because there is a python implementation for base tools, that implemented most of the functionality, and will soon replace the C base tools. In addition, the python implementation is already in use in the azure pipeline.

**Reply by Elemeants** [reply]
Oh, thanks! I didn't knew that.
